splice/models.py

```diff
@@ -95,16 +97,18 @@ class Adgroup(db.Model):
     id = db.Column(db.Integer(), autoincrement=True, primary_key=True, info={"identity": [1, 1]})
     locale = db.Column(db.String(14), nullable=False)
 
-    type = db.Column(db.String(16))
-    paused = db.Column('paused', db.Boolean(), nullable=False, server_default=db.text(u'false'))
+    type = db.Column(db.String(16), nullable=False)
+    paused = db.Column('paused', db.Boolean(), nullable=False, default=False)
     frequency_cap_daily = db.Column(db.Integer())
     frequency_cap_total = db.Column(db.Integer())
-    name = db.Column(db.String(255))
+    name = db.Column(db.String(255), nullable=False)
```

Comment:
We currently have a lot of data in the db that has Adgroup.name == NULL - so this is going to break migrations.  Is it absolutely necessary to have this?